packages/wordpress-external-dependencies-plugin/wp-modules.js

```diff
@@ -0,0 +1,75 @@
+const wpModules = new Map( [
```

Comment:
Why not generate `@wordpress/` deps instead of defining each one manually, since their naming patterns are predictable and there seems to be a kind of contract that they'll remain to be so?

Reply:
My thinking was that there are `@wordpress/` packages that aren't available as script dependencies in WordPress and would need to be bundled. It may be that those packages are not for the browser and this bundler shouldn't need to worry about them.

Reply:
This would be an interesting configuration point, allow consumers to specify the elements of this map so if they're designing a plugin that exposes additional WordPress script dependencies, they can define them here.

Reply:
It very impractical because there will be more packages created and published and you will have to remember to update it.